src/components/Layout.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Header from './Header';
+import Footer from './Footer';
+
+class Layout extends React.Component{
+
+    render(){
+        return(
+            <>
+                <Header />
```

Comment:
This component indent tab space is different from others. Please check this

Reply:
You're right. We decided use 2 spaces. I solve the problem with this component and others.